Runtime/Conditions/PositionalConditions/ObjectInColliderCondition.cs

```diff
@@ -25,6 +25,10 @@ public class EntityData : IObjectInTargetData
             /// </summary>
             [DataMember]
             [DisplayName("Object to collide into")]
+#if CREATOR_PRO
+            [CheckForCollider]
+            [ColliderAreTrigger]
```

Comment:
Now that I think about it, maybe these two can be merged into a single one that receives a parameter 🤔

Reply:
No i dont want this. They are very different internal.